### PR TITLE
Create virtual-onboarding.md

### PR DESCRIPTION
**Pull request by beckigreen**
### Summary

This is a new subchapter in collaboration (creating new communities and teams). 
Coauthors: Aida Mehonic (@AidaMehonic), Isabel Birds (@IsabelBirds), Becki Green (@beckigreen) 

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### List of changes proposed in this PR (pull-request)

New subchapter on virtual onboarding.

### What should a reviewer concentrate their feedback on?

- Any missing topics?
- Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@all-contributors please add @IsabelBirds for ideas

**Comment by malvikasharan** [comment]
@all-contributors please add @beckigreen for ideas

**Comment by malvikasharan** [comment]
@all-contributors please add @AidaMehonic for ideas

**Review by malvikasharan — APPROVED** [review]
Looks great! It can use some examples, but it's ready to merge even without that.
Thank you, everyone! and apology that it took so long to review this.
I am tagging @AidaMehonic and @EKaroune for adding some info from their onboarding work with PDRA and DECOVID onboarding and final review if they can.

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Names and roles are also easy to forget, particularly if we aren't seeing people in the office every day. 
+It can be helpful to store this information somewhere, for example, by sending around an introductory email.
+Also consider organising online social events, such as coffee chats or team activity like a video game or escape room to help new members integrate into the group. 
+
+An onboarding event sharing important information like contact details, organigram, shared calendar, resources available to everyone, organisation policy and social norms can be extremely useful.
+
+## Support
+
+You can provide one-on-one support - such as a buddy for informal peer support, or a mentor, for more formal support.
+This helps the new joiner understand group norms, and allows them to ask any sensitive questions they wouldn't like to raise in a group discussion.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:22]:
```suggestion
This helps the new joiner understand group norms, and allows them to ask any sensitive questions they wouldn't like to raise in a group discussion.
You should consider that the buddy or mentor should remain for a longer period than would be needed with in person onboarding.

Co-working with a buddy or several team members is another informal way to build relationships with new team members and is an informal environment to address any questions that may have arisen in the first few days or weeks of employment. 
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+## Onboarding Document Guidelines
+
+Your project can have an onboarding document or manual.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:26]:
```suggestion
Your project can have an onboarding document, manual or folder of resources. 
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+## Support
+
+You can provide one-on-one support - such as a buddy for informal peer support, or a mentor, for more formal support.
+This helps the new joiner understand group norms, and allows them to ask any sensitive questions they wouldn't like to raise in a group discussion.
+
+## Onboarding Document Guidelines
+
+Your project can have an onboarding document or manual. 
+This creates a set of shared expectations and acts as a reference for new members to find information about resources, or who to ask for help if they have a problem.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:27]:
```suggestion
This creates a set of shared expectations and acts as a reference for new members to find information about resources. It should also contain information about who to ask for help if they have a problem.
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+### Suggested Sections and Content
+
+#### Group Culture
+
+Every group or workplace has a different culture, and expectations aren't always clear when a new member joins. 
+You can help them settle in by making the culture clear here.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:34]:
```suggestion
You can help them settle in by making the culture clear in written form. 
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+#### Group Culture
+
+Every group or workplace has a different culture, and expectations aren't always clear when a new member joins. 
+You can help them settle in by making the culture clear here. 
+
+This can be especially important for people who experience discrimination - do they know that they can approach you and be listened to if they experience harassment or discrimination?
+
+Group culture sections can include:
+* What members of the group can expect from you - turnaround times for feedback, support.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:39]: Who do you mean by you? Leadership group? The project lead? Line managers? I think this needs to be made more clear

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+* What you expect from members of the group - timekeeping, communication. This may vary between different roles.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:40]: Maybe change this to what is expected as part of this team in general (for everyone to do) and then another section of what is expected in certain team roles.

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+* Expectations for working hours.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:41]: I think it is important here to mention flexible working and the fact that this needs to be made explicit from the start so that it is fair for all in the team/workplace.

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+* Equality, inclusion and diversity statement.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:42]: Mention code of conduct in general here

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+* Open science and how you expect the group to practice this.
+
+#### Key Contacts
+
+It can be hard to ask for help, particularly when working remotely - you can't just lean over to the person at the next desk and ask! 
+
+Key contacts could include:
+* Group technicians/managers
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:50]:
```suggestion
* Group technicians/managers.
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+* Key collaborators, and info on what areas of the project they are involved in.
+* IT - include detail - is there a form, or a specific person to email?
+
+#### Communication Channels
+
+- Preferred method of communications, _for example email, video call, slack message_
+- Ensure any communication preferences are ..communicated.. from the beginning
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:57]:
```suggestion
- Ensure any communication preferences are ..communicated.. from the beginning. 
It should be a priority that the new team member is sent all relevant information to join these communications channels and someone should check that they have been able to set them up. 
Offer help with this process and also training on any communications channel that the new team member has not used before so that they can communicate with the other team members quickly after joining. 
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+
+#### Accessing Shared Documents
+
+- What method of sharing is preferred? _for example google docs, RMarkdown, shared documents over email, GitHub, OneDrive_ 
+- Ensure invitations are set up for calendar meetings and access is in place
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:62]:
```suggestion
- Ensure invitations are set up for calendar meetings and access is in place.
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+
+#### Expectations in Meetings
+
+- Individuals may not have adequate access to equipment/internet.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:66]:
```suggestion
- Individuals may not have adequate access to equipment or the internet. 
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+
+- What method of sharing is preferred? _for example google docs, RMarkdown, shared documents over email, GitHub, OneDrive_ 
+- Ensure invitations are set up for calendar meetings and access is in place
+
+#### Expectations in Meetings
+
+- Individuals may not have adequate access to equipment/internet. 
+Discussions about whether any support is required for home setup (for example webcams, microphone, laptop) can allow for any needs to be addressed.
+- Meeting etiquette - learning expectations for different meetings can be overwhelming for a new starter. 
+Clearly outlining expected attendance and contribution can help overcome this, _for example are attendees required to read the papers prior to journal club? Are attendees asked to present their work regularly? Are you expecting videos to be on?_
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:69]:
```suggestion
Clearly outlining expected attendance and contributions can help overcome this, _for example, are attendees required to read the papers prior to journal club? Are attendees asked to present their work regularly? Are you expecting videos to be on?_
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+- Meeting times - consider the circumstances of your group when organising regular meetings - does everyone have the same opportunity to attend? _for example some groups try to arrange Athena SWAN meeting times, whereby meetings occur between 10-3, meaning that those with children are more able to attend this_.
+
+#### Training
+
+On tools, the group uses often - for example GitHub
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:74]:
```suggestion
On tools, the group uses often - for example GitHub and communication channels such as Slack.
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+
+* Use Zenodo or similar to host training materials for easier sharing
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:76]:
```suggestion
* Use [Zenodo](https://zenodo.org/), or a similar open repository, to host training materials for easier sharing. 
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+- Meeting etiquette - learning expectations for different meetings can be overwhelming for a new starter. 
+Clearly outlining expected attendance and contribution can help overcome this, _for example are attendees required to read the papers prior to journal club? Are attendees asked to present their work regularly? Are you expecting videos to be on?_
+- Meeting times - consider the circumstances of your group when organising regular meetings - does everyone have the same opportunity to attend? _for example some groups try to arrange Athena SWAN meeting times, whereby meetings occur between 10-3, meaning that those with children are more able to attend this_.
+
+#### Training
+
+On tools, the group uses often - for example GitHub
+
+* Use Zenodo or similar to host training materials for easier sharing 
+* Use a Youtube channel to store induction & training for people to watch in their own time
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:77]:
```suggestion
* Use a Youtube channel to store induction & training material for people to watch in their own time.
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 17]
```diff
@@ -0,0 +1,98 @@
...
+* Consider running training & induction sessions every 2-3 months as new people join
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:78]:
```suggestion
* Consider running training & induction sessions every 2-3 months as new people join.
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 18]
```diff
@@ -0,0 +1,98 @@
...
+* Consider a Slack channel for new joiners - for questions and discussion
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:79]:
```suggestion
* Consider a Slack channel for new joiners - for questions and discussion. 
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 19]
```diff
@@ -0,0 +1,98 @@
...
+
+#### Training
+
+On tools, the group uses often - for example GitHub
+
+* Use Zenodo or similar to host training materials for easier sharing 
+* Use a Youtube channel to store induction & training for people to watch in their own time
+* Consider running training & induction sessions every 2-3 months as new people join
+* Consider a Slack channel for new joiners - for questions and discussion 
+* Consider providing Office Hours to support new joiners using new tools - for example, GitHub
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:80]:
```suggestion
* Consider providing Office Hours to support new joiners using new tools - for example, GitHub.
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 20]
```diff
@@ -0,0 +1,98 @@
...
+* Get feedback on your training
+* Try and make your training session as interactive as possible with exercises
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:82]:
```suggestion
* Try to make your training sessions as interactive as possible with exercises and discussions.
* Get feedback on your training and use this to adjust and improve your resources.
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+* Consider running training & induction sessions every 2-3 months as new people join
+* Consider a Slack channel for new joiners - for questions and discussion 
+* Consider providing Office Hours to support new joiners using new tools - for example, GitHub
+* Get feedback on your training
+* Try and make your training session as interactive as possible with exercises 
+
+#### Resources 
+
+* Link to the group website/Twitter
+* Links to other resources - institution wellbeing services.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:87]:
```suggestion
* Link to the group website.
* Link to group social media accounts - Twitter, Facebook, Instagram.
* Links to other resources - institution wellbeing services.
* Links to group policies.
```

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 22]
```diff
@@ -0,0 +1,98 @@
...
+
+Joining a new group virtually is a very different experience from joining in person, and it can be hard to know if someone is struggling or isolated.  
+You can help to make this experience less overwhelming by having a clear onboarding plan and resources. 
+
+## Introducing the Team
+
+It can be intimidating to be the first to speak as a new member - consider asking your team to do a "who's who", and to introduce themselves first in meetings.
+
+Names and roles are also easy to forget, particularly if we aren't seeing people in the office every day. 
+It can be helpful to store this information somewhere, for example, by sending around an introductory email.
```
Comment (EKaroune) [inline book/website/collaboration/new-community/new-community-onboarding.md:14]:
```suggestion
It can be helpful to store this information somewhere, for example, by sending around an introductory or onboarding email.
```

**Review by EKaroune — APPROVED** [review]
This is looking good but have a look at my suggested edits. 😀

**Comment by Arielle-Bennett** [comment]
Hi all, I'm currently working on a piece on the overall ethos behind a good onboarding process (as part of some internal project work I'm doing). I was wondering if we could hit the next collaboration cafe to discuss structuring (to remove overlap) and get both of them online?

**Comment by malvikasharan** [comment]
Thank you @EKaroune for reviewing this.
I am pinging @beckigreen and @IsabelBirds to check if they would like to come along to next week's collaboration cafe to discuss this chapter with @Arielle-Bennett.
Potentially, this chapter can be teased out as a standalone chapter if you would like to create a background (ethos) piece that can go before this. I am happy to chat and co-work on this next week.

**Comment by beckigreen** [comment]
Thanks @malvikasharan & @EKaroune ! Sounds great @Arielle-Bennett, I have meetings until 4 but would love to join you after if that's not too disruptive?

**Comment by Arielle-Bennett** [comment]
> Thanks @malvikasharan & @EKaroune ! Sounds great @Arielle-Bennett, I have meetings until 4 but would love to join you after if that's not too disruptive?

Not disruptive at all! Looking forward to it :)

**Comment by IsabelBirds** [comment]
@Arielle-Bennett looks like I might not make the co-working call because of a conference, but happy to be assigned a few tasks depending on what plans come out of the call if I don't make it :)

**Comment by Arielle-Bennett** [comment]
No problem @IsabelBirds! We'll let you know (I'm envisioning we mainly agree on overall chapter structure and check that any content overlap makes sense rather than duplication). If you have time, please do check out my HackMD draft here: https://hackmd.io/@iXOo9P13RRq_GsyrM8bKsw/BJL2mQk2_/edit

**Comment by malvikasharan** [comment]
@beckigreen @IsabelBirds and @Arielle-Bennett Going through Arielle's draft and the chapter here, I see a lot of overlap but also a potential to generalise this chapter for both virtual and in-person onboarding. Particularly, the background and motivation written by Arielle can go to the landing page. The rest of her draft and content in this chapter can be discussed to spread out to cover virtual and in-person topics. I can imagine that there will be overlap in there but maybe a good place to start in tomorrow's collaboration cafe?

To summarise, I am proposing this structure:
```
  - title: Onboarding New Members
    file: collaboration/onboarding      
    - title: Virtual Onboarding
      file: collaboration/onboarding/onboarding-virtual
    - title: In-person Onboarding
      file: collaboration/onboarding/onboarding-inperson
```

A lot of this chapter and Arielle's draft overlap (commented in the HackMD).

**Comment by malvikasharan** [comment]
Subchapter suggestion based on the discussion:
- Important Aspects of onboarding
- Planning and process
- Consideration for in-person and virtual

File: book/website/collaboration/new-community/new-community-onboarding.md [hunk 23]
```diff
@@ -0,0 +1,98 @@
...
+# Virtual Onboarding of New Members
+
+## Summary
+
+Joining a new group virtually is a very different experience from joining in person, and it can be hard to know if someone is struggling or isolated.  
+You can help to make this experience less overwhelming by having a clear onboarding plan and resources. 
+
+## Introducing the Team
+
+It can be intimidating to be the first to speak as a new member - consider asking your team to do a "who's who", and to introduce themselves first in meetings.
```
[the remark this answers is not in the document]
Comment (beckigreen) [reply book/website/collaboration/new-community/new-community-onboarding.md]: I have removed this text - thank you Emma

File: book/website/collaboration/new-community/new-community-onboarding.md (hunk 6 above)
Comment (beckigreen) [reply book/website/collaboration/new-community/new-community-onboarding.md]: I have merged these two together and positioned it from the lens of the team in general - thank you!

File: book/website/collaboration/new-community/new-community-onboarding.md (hunk 8 above)
Comment (beckigreen) [reply book/website/collaboration/new-community/new-community-onboarding.md]: Done!

File: book/website/collaboration/new-community/new-community-onboarding.md (hunk 7 above)
Comment (beckigreen) [reply book/website/collaboration/new-community/new-community-onboarding.md]: Absolutely - thanks for pointing this out. I've included some mention of this and we will also be covering this more when restructuring the chapter and sub chapters!

**Comment by malvikasharan** [comment]
@all-contributors please add @Iain-S for review

**Comment by jcolomb** [comment]
Hello, it would be nice to see interactions with  https://github.com/shreyadimri/the-turing-way/blob/main/book/website/collaboration/team-manual.md 
are there doubloons, does it needs cross linking ? ...
